pandas/core/categorical.py

```diff
@@ -1862,8 +1990,11 @@ def convert_categorical(x):
     if get_dtype_kinds(to_concat) - set(['object', 'category']):
         # convert to object type and perform a regular concat
         from pandas.core.common import _concat_compat
-        return _concat_compat([np.array(x, copy=False, dtype=object)
-                               for x in to_concat], axis=0)
```

Comment:
This was a YAPF thing.  Clearly it likes moving everything to the next line.  Having an additional check for open parenthesis on the line would point out a lot of these that I missed through my manual go through.  Or this might be the ticket:

```
SPLIT_PENALTY_AFTER_OPENING_BRACKET
    The penalty for splitting right after the opening bracket.
```
I can play around with that a bit.